### DISCUSSION TITLE
Using btc_city_fnc_cleanUp to clean base

### DISCUSSION THREAD

**Discussion by SgtKrunch** [Vdauphin/HeartsAndMinds#1403] [Question and answer]
As the title states, I used to use that function to clean up the base whenever players left the base after dropping stuff on the ground. Is there a way to still do this? my old trigger doesn't seem to work anymore and I can't really figure out why.

**Comment by Vdauphin** [comment]
Hello,
Yes it is working
H&M use it everytime you left a city
Cheers

**Comment by SgtKrunch** [comment]
```
[[btc_base]] call btc_fnc_city_cleanUp;
if (btc_debug) then {
    ["Base Cleanup Triggered."] remoteExec ["systemChat", 0];
};
```
I used to use this in a trigger that activated whenever players were no longer in the trigger area. It doesn't seem to work anymore. I've tried to fix it to no avail. Any idea how I could fix this?

**Reply by Vdauphin** [reply]
Try `btc_city_fnc_cleanUp`?
See https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/city/cleanUp.sqf

**Reply by SgtKrunch** [reply]
I have the function called in my trigger, did the function change semi-recently? It use to work when I called it with a trigger. Do I need to wait till players leave the 500x500 area defined in the function?

**Reply by Vdauphin** [reply]
> did the function change semi-recently? 

Check the commit history

> Do I need to wait till players leave the 500x500 area defined in the function?

It depend what is in `_playableUnits`

**Reply by SgtKrunch** [reply]
I think I see what the issue is and I'm going to feel really dumb if it's what I think it is lol. I'll update when I'm able to test. Thank you for your patience so far.
